From e431ab003f3fabf616b6eb7dc93f8967bc9473e5 Mon Sep 17 00:00:00 2001
From: Milan Crha <mcrha@redhat.com>
Date: Wed, 22 Nov 2023 09:44:40 +0100
Subject: [PATCH] gs-appstream: Remove use of AS_PROVIDED_KIND_PYTHON_2

It's dropped in appstream 1.0.0 and it's not used anywhere in the gnome-software
code, thus just remove it, rather than have it only for pre-1.0.0 appstream version.

Closes https://gitlab.gnome.org/GNOME/gnome-software/-/issues/2388

Upstream-Status: Backport [https://gitlab.gnome.org/GNOME/gnome-software/-/commit/e431ab003f3fabf616b6eb7dc93f8967bc9473e5]
---
 lib/gs-appstream.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/lib/gs-appstream.c b/lib/gs-appstream.c
index 6504d6f25..4fd7f5334 100644
--- a/lib/gs-appstream.c
+++ b/lib/gs-appstream.c
@@ -587,8 +587,6 @@ gs_appstream_refine_add_provides (GsApp *app, XbNode *component, GError **error)
 					kind = AS_PROVIDED_KIND_FIRMWARE_RUNTIME;
 				else if (g_strcmp0 (fw_type, "flashed") == 0)
 					kind = AS_PROVIDED_KIND_FIRMWARE_FLASHED;
-			} else if (g_strcmp0 (element_name, "python2") == 0) {
-				kind = AS_PROVIDED_KIND_PYTHON_2;
 			} else if (g_strcmp0 (element_name, "python3") == 0) {
 				kind = AS_PROVIDED_KIND_PYTHON;
 			} else if (g_strcmp0 (element_name, "dbus") == 0) {
-- 
GitLab

